proxy/src/main/java/net/md_5/bungee/BungeeCord.java

```diff
@@ -180,7 +180,14 @@ public BungeeCord() throws IOException
         // Java uses ! to indicate a resource inside of a jar/zip/other container. Running Bungee from within a directory that has a ! will cause this to muck up.
         Preconditions.checkState( new File( "." ).getAbsolutePath().indexOf( '!' ) == -1, "Cannot use BungeeCord in directory with ! in path." );
 
-        System.setSecurityManager( new BungeeSecurityManager() );
+        try
```

Comment:
PRs like this are always difficult to handle because they are better suited to a bug report. Nonetheless my view on what should be done if you want this pulled:

* Watch formatting (space before closing brace)
* Move setting of security manager to after system streams have been replaced and logger initialized
* Change message to logger.log(Level.warn, "Could not set security manager (currently {0})",System.getSecurityManager()); or similar, currently its a bit misleading.